fastboot.jinja2

```diff
@@ -0,0 +1,179 @@
+
+
+
+{% block context %}
+  test_character_delay: 10
+{% endblock context %}
+
+{% block protocols %}
+protocols:
+  lava-lxc:
+    name: lxc-target
+    template: debian
+    distribution: debian
+    release: buster
+    arch: amd64
+    mirror: http://deb.debian.org/debian
+{% endblock protocols %}
+
+{% block actions %}
+{% block lxc %}
+- deploy:
+    namespace: tlxc
+    timeout:
+      minutes: {{ lxc_deploy_timeout }}
+    to: lxc
+    packages:
+    - wget
+    - unzip
+{% if install_packages == true %}
+    - abootimg
+    - cpio
+    - curl
+    - e2fsprogs
+    - file
+    - git
+    - gzip
+    - img2simg
+    - libguestfs-tools
+    - mkbootimg
+    - mktemp
+    - simg2img
+    - xz-utils
+{% endif %}
+    os: debian
+
+- boot:
+    namespace: tlxc
+    prompts:
+    - 'root@(.*):/#'
+    timeout:
+      minutes: {{ lxc_boot_timeout }}
+    method: lxc
+
+{% if install_fastboot == true %}
+- test:
+    namespace: tlxc
+    timeout:
+      minutes: {{ lxc_install_fastboot_timeout }}
+    definitions:
+    - repository: https://github.com/Linaro/test-definitions.git
+      from: git
+      path: automated/linux/android-platform-tools/install.yaml
+      name: install-android-platform-tools-r2600
+      parameters:
+        LINK: 'https://dl.google.com/android/repository/platform-tools_r26.0.0-linux.zip'
+{% endif %}
+{% endblock lxc %}
+
+{% block deploy_target %}
+- deploy:
+    timeout:
+      minutes: {{ target_deploy_timeout }}
+    to: {{ deploy_target }}
+    namespace: target
+    images:
+{% if ptable == true %}
+      ptable:
+        url: {{PTABLE_URL}}
+{% if PTABLE_URL_COMP is defined %}
+        compression: {{PTABLE_URL_COMP}}
+{% endif %}
+{% endif %}
+{% if boot == true %}
+      {{ boot_label }}:
+        url: {{BOOT_URL}}
+{% block kernel_extra_args %}
+{% endblock kernel_extra_args %}
+{% if BOOT_URL_COMP is defined %}
+        compression: {{BOOT_URL_COMP}}
+{% endif %}
+{% if apply_overlay == "bootimg" %}
+        apply-overlay: true
+{% endif %}
+{% block boot_extra_args %}
+{% endblock boot_extra_args %}
+{% endif %}
+{% if DTB_URL is defined %}
+      dtb:
+        url: {{DTB_URL}}
+{% endif %}
+{% if MODULES_URL is defined %}
+      modules:
+        url: {{MODULES_URL}}
+{% if MODULES_URL_COMP is defined %}
+        compression: {{MODULES_URL_COMP}}
+{% endif %}
+{% endif %}
+{% if rootfs == true %}
+      {{ rootfs_label }}:
+        url: {{ROOTFS_URL}}
+{% if ROOTFS_URL_COMP is defined %}
+        compression: {{ROOTFS_URL_COMP}}
+{% endif %}
+{% if apply_overlay == "rootfs" %}
+        apply-overlay: true
+{% endif %}
+{% block rootfs_extra_args %}
+{% endblock rootfs_extra_args %}
+{% endif %}
+    os: {{DEPLOY_OS}}
+{% endblock deploy_target %}
+
+{% block boot_target %}
+- boot:
+    namespace: target
+{% if auto_login == true %}
+    auto_login:
+      login_prompt: "{{ auto_login_prompt }}"
+      username: "{{ auto_login_username }}"
```

Comment:
we should support optional password section here. Not mandatory for now but nice to have

Reply:
OK